src/Hyperion/Analysis.hs

```diff
@@ -49,11 +49,13 @@ namesOf = go []
     coerce = contramap (const ()) . fmap (const ())
 
 analyze
-  :: Text -- ^ Benchmark name
-  -> Sample -- ^ Measurements
+  :: Text -- ^ Executable name.
+  -> Text -- ^ Benchmarks name.
+  -> Text -- ^ Benchmark name.
+  -> Sample -- ^ Measurements.
   -> Report
-analyze name samp = Report
-    { _reportBenchName = name
+analyze executableName benchesName name samp = Report
+    { _reportBenchName = Text.concat [executableName, ".", benchesName, ".", name]
```

Comment:
So when we added the Text argument to `defaultMain` it was for the user to specify the package right ?

We would thus want the following naming: `$packageName.$exectuableName.json` right ?